manifests/overlays/apps/apps-addon/starburst/starburst-galaxy-quickstart.yaml

```
@@ -57,22 +57,22 @@ spec:
         success: You have all of the connection details you need to connect to your Starburst Galaxy cluster.  Proceed to the next step.
         failed: Try the steps again.
 
-    - title: Launch JupyterHub from Red Hat OpenShift AI
+    - title: Launch Jupyter from Red Hat OpenShift AI
       description: |-
-        ### To launch a JupyterHub notebook server from within the Red Hat OpenShift AI console:
+        ### To launch a Jupyter notebook server from within the Red Hat OpenShift AI console:
         1. Click **Applications** &#x2192; **Enabled**.
-        2. Find the **JupyterHub** card.
-        3. Click **Launch** on the JupyterHub card to access the JupyterHub **Start a notebook server** page.  A new browser tab will open displaying the **Start a notebook server** page.
+        2. Find the **Jupyter** card.
+        3. Click **Launch** on the Jupyter card to access the Jupyter **Start a notebook server** page.  A new browser tab will open displaying the **Start a notebook server** page.
         4. Select the **Standard Data Science** notebook image and the **Default** container size.
         5. Add the environment variables for **TRINO\_USERNAME**, **TRINO\_PASSWORD**, **TRINO\_HOSTNAME**, and **TRINO\_PORT** using the connection details you obtained in the previous task.
-        6. Click on **Start server** to launch JupyterHub.
+        6. Click on **Start server** to launch Jupyter.
 
         If you have not previously authorized the jupyterhub-hub service account to access your account, the **Authorize Access** page appears prompting you to provide authorization. Inspect the permissions selected by default, and click the **Allow selected permissions** button.
```

```suggestion
        If you have not previously authorized the `jupyter-hub` service account to access your account, the **Authorize Access** page appears prompting you to provide authorization. Inspect the permissions selected by default, and click the **Allow selected permissions** button.
```

Two suggestions:
1. Remove "hub" from the service account name?
2. Enclose the service name in backticks for consistency with OpenShift doc convention?

---

The service name is provided by a third party and is not referring to an actual proper name of something - we need to leave it that way until we hear from the Starburst team.  We should not make this change now.